src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -284,41 +284,80 @@ def makeSubscriptions(self, workflow):
             #    but an iterator of length 'stride' and then we should be doing:
             #    for workflow in workflows:
             if isinstance(workflow, MSOutputTemplate):
-                if workflow['isRelVal']:
-                    pass
-                else:
-                    try:
+                ddmReqList = []
+                try:
+                    if workflow['isRelVal']:
+                        for dMap in workflow['destinationOutputMap']:
+                            try:
+                                ddmRequest = DDMReqTemplate('copy',
+                                                            item=dMap['datasets'],
+                                                            n=workflow['numberOfCopies'],
+                                                            site=dMap['destination'],
+                                                            group='RelVal')
+                            except KeyError as ex:
```

Comment:
I would actually put a check for it, instead of assuming that the KeyError comes from the lack of value for the `site` attribute. Unless, the only possibility to raise such KeyError would come from that.

Reply:
I implicitly made it to be the case with [1]. There is only one other place where a `KeyError`  is generated [2], but we can never hit it because we generate the keys in this call by hand, only the values are taken from the document but they may generate only `TypeError`.

[1]
https://github.com/dmwm/WMCore/blob/296b118889cb05ad35e486f4873b56b4d1c7168d/src/python/WMCore/Services/DDM/DDM.py#L105

[2]
https://github.com/dmwm/WMCore/blob/296b118889cb05ad35e486f4873b56b4d1c7168d/src/python/WMCore/Services/DDM/DDM.py#L85